src/commands/tsgen.ts

```diff
@@ -1,4 +1,5 @@
-import {Command, flags} from '@contentstack/cli-command'
+import {Command} from '@contentstack/cli-command'
+import {flags as _flags} from '@contentstack/cli-utilities'
```

Comment:
@aman19K Basically `_ + variableName (_flags)` is used to tell the programming/lint/ts that it is an unused variable so ignore the lint. So try to use different alias. It's not critical but in terms of best practice, it could be good to follow the proper pattern.